test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Util/Util.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.brokers;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.languages;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.apps.type;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.entity;
+using Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.queue;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Util
+{
+	public static class Utils
+	{
+		public static string GetEnvVariable(string varName) { return Environment.GetEnvironmentVariable(varName); }
+		public static string BuildCloudBrokerName(QueueType queueType, AppType appType, Language language)
+		{
+			//return Constants.
```

Comment:
can we remove this comment if not required ??

Reply:
Removed.